[doc] deepspeed universal checkpoint (#35015)
huggingface/transformers: docs/source/en/deepspeed.md

}
```

> ### Universal checkpointint

Reviewer: ```suggestion
### Universal Checkpointing
```

> DeepSpeed [Universal Checkpointing](https://www.deepspeed.ai/tutorials/universal-checkpointing) feature is a powerful tool for saving and loading model checkpoints in a way that is both efficient and flexible, enabling seamless model training continuation and finetuning across different model architectures, different parallelism techniques and training configurations.

Reviewer: ```suggestion
[Universal Checkpointing](https://www.deepspeed.ai/tutorials/universal-checkpointing) is an efficient and flexible feature for saving and loading model checkpoints. It enables seamless model training continuation and fine-tuning across different model architectures, parallelism techniques, and training configurations.
```

> You can resume training with universal checkpoint by setting [`load_universal`](https://www.deepspeed.ai/docs/config-json/#checkpoint-options) to `true` in the config file.

Reviewer: ```suggestion
Resume training with a universal checkpoint by setting [load_universal](https://www.deepspeed.ai/docs/config-json/#checkpoint-options) to `true` in the config file.
```
